readme.md

```diff
@@ -88,24 +127,47 @@ The default export is `remarkGemoji`.
 
 ### `unified().use(remarkGemoji)`
 
-Plugin to turn Gemoji shortcodes into emoji.
+Plugin to turn gemoji shortcodes (`:+1:`) into emoji (`👍`).
+There are no options.
+
+## Syntax
+
+This plugin looks for the regular expression `/:(\+1|[-\w]+):/g` in text in
+markdown (excluding code and such).
+If the value between the two colons matches a know gemoji shortcode, then its
+replaced by the corresponding emoji.
```

Comment:
Not sure where this would fit, but would it make sense to have some notes on how this works with remark-directive?
There have been questions on interactions between the two in the past https://github.com/remarkjs/remark/discussions/651

Reply:
That conversation did result in a change in `remark-directive` though, which means they no longer overlap.
It’s mentioned in the [syntax section on `micromark-extension-directive`](https://github.com/micromark/micromark-extension-directive#syntax) (search for emoji).
So, as they don’t overlap, does it need to be mentioned here?

Reply:
Cool beans, fine by me